app/src/main/res/values/strings.xml

```diff
@@ -787,4 +787,7 @@
     <string name="thermometer_bottom_sheet_text">Thermometer instrument is used to measure ambient temprature. It can be measured using inbuilt ambient temprature sensor or through SHT21.</string>
     <string name="thermometer_bottom_sheet_desc">Thermometer instrument is used to measure ambient temprature. It can be measured using inbuilt ambient temprature sensor or through SHT21.</string>
     <string name="no_thermometer_sensor">Device does not have Thermometer Sensor</string>
+    <string name="thermo_unit">F</string>
```

Comment:
We'll use Celcius

Reply:
And let user select which unit want from configurations

Reply:
Yes , I am going to add that functionality . I just wanted to confirm it works or not till here.

Reply:
I was referring to units here. Throughout the app we are using Metric units, not imperial